BrowserKit/Sources/WebEngine/WKWebview/Policy/WKPolicyDecider.swift

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import WebKit
+
+/// An object that decides the policy for a navigation request by `WebKit`'s delegate methods.
+protocol WKPolicyDecider {
+    /// The next decider in the chain, that can eventually respond with a `WKPolicy`
+    /// when this decider is not able to handle it.
+    var next: WKPolicyDecider? { get set }
```

Comment:
I stumbled on the protocol after, and I think this would've help me understand the other parts of the code with this small name change
```suggestion
    var nextDecider: WKPolicyDecider? { get set }
```

Reply:
yes agree it's easier to read with this small change